### PR TITLE
Add get started

### LINKED ISSUE #79: Missing get_started.md
A `get_started.md` should be added to this SDK to display a get started procedure in the Astarte documentation page.
See the python [get started page](https://docs.astarte-platform.org/device-sdks/common/get_started/python.html) the relative [get_started.md](https://github.com/astarte-platform/astarte-device-sdk-python/blob/master/get_started.md) and the [github workflow](https://github.com/astarte-platform/astarte-device-sdk-python/blob/master/.github/workflows/doc-workflow.yaml).

### PR DESCRIPTION
**Pull request by osmanhadzic**
- Add GET_STARTED.md file to provide a quick start guide for new users.
- Update the documentation workflow to include a comparison and substitution for GET_STARTED.md, ensuring it’s part of the documentation build and review process.

This `GET_STARTED.md` guide will make it easier for new users to quickly understand how to set up and start using the project. Integrating it into the documentation workflow helps keep it up-to-date and aligned with any future changes in the project.

Closes #79

### CODE REVIEW

File: GET_STARTED.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
+<!---
+  Copyright 2022 SECO Mind Srl
```
Comment (harlem88) [inline GET_STARTED.md:2]:
```suggestion
  Copyright 2024 SECO Mind Srl
```

**Review by harlem88 — CHANGES_REQUESTED** [review]

File: .github/workflows/docs-workflow.yaml [hunk 2]
```diff
@@ -65,3 +65,116 @@ jobs:
       working-directory: ./docs
       run: |
         git push origin master
+  get-started-docs:
+    runs-on: ubuntu-latest
+    steps:
```
Comment (harlem88) [inline .github/workflows/docs-workflow.yaml:70]:
```suggestion
    needs: [api-docs]
    steps:
```

**Review by harlem88 — CHANGES_REQUESTED** [review]

File: .github/workflows/docs-workflow.yaml (hunk 2 above)
Comment (sorru94) [reply .github/workflows/docs-workflow.yaml:70]: The two workflows can be executed independently, so I would not bind them with a `needs`.

File: GET_STARTED.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+```
+Or in a deterministic way:
+
+``` CSharp
+Guid nameSpace = Guid.Parse("f79ad91f-c638-4889-ae74-9d001a3b4cf8");
+string macAddress = "98:75:a8:0d:96:db";
+deviceId = AstarteDeviceIdUtils.GenerateId(nameSpace, macAddress);
+```
+# Registering a device
+
```
Comment (sorru94) [inline GET_STARTED.md:30]:
```suggestion
```
Be consistent with the number of lines after and before the headings.

File: GET_STARTED.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+while (true)
+{
+    double value = Random.Shared.NextDouble();
+    Console.WriteLine("Streaming value: " + value);
+
+    await valuesInterface.StreamData($"/{sensorUuid}/value", value, DateTime.Now);
+
+    Thread.Sleep(1000);
+}
+
```
Comment (sorru94) [inline GET_STARTED.md:88]: Missing ```

File: GET_STARTED.md [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+### Using a registered device
+
+### Run the code from the root of the example project with
+
+``` bash
+dotnet run AstarteDeviceSDKExample.csproj --r "<realm>"  --p "<pairing-url>" --d "<device-id>" --c "<credential-secret>"
+```
+
+where `pairing-url` is the URL to reach Pairing API in your Astarte instance, usually `https://api.<your-astarte-domain>/pairing`.
```
Comment (sorru94) [inline GET_STARTED.md:235]: No need to explain how to run an example in the get started. The get started guide is intended to provide step by step snippets to create your own example.

**Review by sorru94 — APPROVED** [review]

**Review by harlem88 — APPROVED** [review]

**Pull request merged**